### PR TITLE
add support for "stealable glow"

### PR DESCRIPTION
**Pull request by mrbuds**

### CODE REVIEW

File: WeakAuras/Types.lua [hunk 1]
```diff
@@ -3222,6 +3222,7 @@ LSM:Register("statusbar", "Stripes", [[Interface\AddOns\WeakAuras\Media\Textures
 LSM:Register("statusbar", "Thick Stripes", [[Interface\AddOns\WeakAuras\Media\Textures\Statusbar_Stripes_Thick]])
 LSM:Register("statusbar", "Thin Stripes", [[Interface\AddOns\WeakAuras\Media\Textures\Statusbar_Stripes_Thin]])
 LSM:Register("border", "Drop Shadow", [[Interface\AddOns\WeakAuras\Media\Textures\Border_DropShadow]])
+LSM:Register("border", "Stealable Glow", [[Interface\TargetingFrame\UI-TargetingFrame-Stealable]])
```
Comment (InfusOnWoW) [inline WeakAuras/Types.lua:3225]: this doesn't appear to work. That is the resulting border looks wrong.

**Review by InfusOnWoW — COMMENTED** [review]
I'm fine with adding this, though it's a rather minimal prefernce for adding it.

File: WeakAuras/Types.lua (hunk 1 above)
Comment (mrbuds) [reply WeakAuras/Types.lua:3225]: ho right, i added it as a test and forget to remove later

**Comment by mrbuds** [comment]
> I'm fine with adding this, though it's a rather minimal prefernce for adding it.

I'm not convinced it's good enough & generic to be added, as it look bad on anything that is not square & tiny

**Comment by mrbuds** [comment]
i closed https://github.com/Stanzilla/LibCustomGlow/pull/28

**Pull request closed without merging**